solver/pb/ops.pb.go

```diff
@@ -54,7 +54,30 @@ func (x NetMode) String() string {
 	return proto.EnumName(NetMode_name, int32(x))
 }
 func (NetMode) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_ops_8d64813b9835ab08, []int{0}
+	return fileDescriptor_ops_751fcd5ed8ef51ef, []int{0}
+}
+
+type SecurityMode int32
+
+const (
+	SecurityMode_CONFINED   SecurityMode = 0
+	SecurityMode_UNCONFINED SecurityMode = 1
+)
+
+var SecurityMode_name = map[int32]string{
+	0: "CONFINED",
+	1: "UNCONFINED",
```

Comment:
@justincormack Would you be okay calling `unconfined` `insecure` instead? I know all security profiles call it unconfined, but the user intent feels clearer with `insecure`.

Reply:
Yes I think that makes sense, although confined vs insecure is kind of weird, and I don't think we should call it secure vs insecure...

Reply:
@justincormack `sandbox`/`sandboxed` ? I'm not too bothered by having both `confined` and `insecure`, because those who will see `confined` will probably have read about its technical meaning, whereas `insecure` should mean something strong to those who won't read much about it.

Reply:
@justincormack told me offline he prefers `confined` and `insecure`.

@kunalkushwaha would you mind updating `UNCONFINED` to `INSECURE` everywhere in this PR? We can keep the word `CONFINED`.

Reply:
btw, I like sandbox/insecure pairing

Reply:
ok with sandbox/insecure too.

Reply:
@tiborvass ^^

Reply:
Perfect let’s roll with sandbox/insecure then. Do you mind updating @kunalkushwaha

Reply:
I will update with sandbox/insecure.